app/models/comment.rb

```diff
@@ -0,0 +1,4 @@
+class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
```

Comment:
Should we use polymorphism instead of having two parent dependencies? For example something like this: https://guides.rubyonrails.org/association_basics.html#polymorphic-associations. Would make it easier to manipulate later on if we have to.

Reply:
I think it's fine to not use polymorphism here since the dependencies are fairly simple

Reply:
Sure that makes sense then. Thanks for clarifying! Will close this and the other comment.